core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -82,6 +85,15 @@ class GroupMetadataManager(brokerId: Int,
    * We use this structure to quickly find the groups which need to be updated by the commit/abort marker. */
   private val openGroupsForProducer = mutable.HashMap[Long, mutable.Set[String]]()
 
+  /* setup metrics*/
+  private val metricConfig: MetricConfig = new MetricConfig().samples(1)
+  val partitionLoadSensor = metrics.sensor("GroupLoadTime", metricConfig)
```

Comment:
Since we use `metrics` object created by KafkaServer, it uses the reporter created by KafkaServer as well, which reports metrics with `kafka.server` prefix. If we do want 'kafka.coordinator.group` prefix for these metrics, we will need to create another reporter. I wonder if it is better to just use `kafka.server` prefix, so the metric is ``kafka.server:type=group-metadata-manager-metrics` with attributes: group-load-time-max, group-load-time-avg.

Reply:
I agree, it would be better to keep it all on one reporter. I'll change this.

Reply:
Consensus:
`kafka.server:type=transaction-coordinator-metrics` with attributes: `partition-load-time-max` and `partition-load-time-avg`
`kafka.server:type=group-coordinator-metrics` with attributes: `partition-load-time-max` and `partition-load-time-avg`